app/components/image/image.html

```diff
@@ -167,3 +167,43 @@
     </rd-widget>
   </div>
 </div>
+
+<div class="row" ng-if="history.length > 0">
+  <div class="col-lg-12 col-md-12 col-xs-12">
+    <rd-widget>
+      <rd-widget-header icon="fa-clone" title="Image layers"></rd-widget-header>
+      <rd-widget-body classes="no-padding">
+        <table id="image-layers" class="table">
+          <thead>
+            <tr>
+              <td>Size</td>
+              <td>Layer</td>
+              <td></td>
+            </tr>
+          </thead>
+          <tbody>
+            <tr ng-repeat="layer in history">
+              <td class="col-md-1" style="white-space:nowrap;">{{layer.Size|humansize}}</td>
+              <td class="col-md-6">
+                <div ng-if="layer.CreatedBy.length > 100">
+                  <span id="layer-command-{{$index}}-full" style="display: none">
+                    {{layer.CreatedBy}}
+                  </span>
+                  <span id="layer-command-{{$index}}-short">
+                    {{layer.CreatedBy|limitTo:100}}{{layer.CreatedBy.length > 100 ? '...' : ''}}
+                  </span>
+                </div>
+                <div ng-if="layer.CreatedBy.length <= 100">
+                  <span id="layer-command-{{$index}}-full">
+                    {{layer.CreatedBy}}
+                  </span>
+                </div>
+              </td>
+              <td class="expand"><div ng-if="layer.CreatedBy.length > 100"><a id="layer-command-expander{{$index}}" class="btn" ng-click='toggleLayerCommand($index)'><span class="glyphicon glyphicon-plus-sign"></span></a></div></td>
```

Comment:
I'd rather see this close to the truncated text than inside a new `<td>`. Can you add it in the previous `<td>` element?